app/ports/templates/ports/portdetail.html

```diff
@@ -9,7 +9,7 @@
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
 {% endblock %}
 
-{% block title %}{{ req_port.name }}- Port |{% endblock %}
+{% block title %}Install {{ req_port.name }} on macOS with{% endblock %}
```

Comment:
Wasn't this sentence supposed to end with `MacPorts`?

Reply:
> Wasn't this sentence supposed to end with `MacPorts`?

`MacPorts` is added automatically at the end by the layout file.